src/Avalonia.Themes.Default/ProgressBar.xaml

```diff
@@ -22,42 +33,49 @@
   </Style>
   <Style Selector="ProgressBar:horizontal">
     <Setter Property="MinWidth" Value="200"/>
-    <Setter Property="MinHeight" Value="14"/>
+    <Setter Property="MinHeight" Value="16"/>
   </Style>
   <Style Selector="ProgressBar:vertical">
-    <Setter Property="MinWidth" Value="14"/>
+    <Setter Property="MinWidth" Value="16"/>
     <Setter Property="MinHeight" Value="200"/>
   </Style>
+  <Style Selector="ProgressBar[Orientation=Vertical] /template/ LayoutTransformControl#PART_LayoutTransformControl">
```

Comment:
The other selectors use the `:vertical` pseudo-class, so best to be consistent here.